service-unavailable/index.html

```diff
@@ -65,13 +65,16 @@ <h1 class="govuk-heading-l">Sorry, this service is unavailable</h1>
       <div class="govuk-width-container ">
         <div class="govuk-footer__meta">
           <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
-          
+
             <h2>Contact us</h2>
-            <p>
-              Email: DigitalService.TradeFinance@ukexportfinance.gov.uk<br>
-              Phone: +44 (0)207 271 8010<br>
-              Opening times: Monday to Friday, 9am to 5pm (excluding public holidays)
-            </p>
+            <dl class="contact-details">
+              <dt>Email:</dt>
+              <dd>DigitalService.TradeFinance@ukexportfinance.gov.uk</dd><br>
```

Comment:
```suggestion
              <dd>DigitalService.TradeFinance@ukexportfinance.gov.uk<br></dd>
```

Reply:
I checked example and maybe our case is different.
I removed `<br>` and added styling (new line) using css.